static/lang/re-en.json

```diff
@@ -2682,6 +2682,9 @@
                 "Reflex": "When you roll a success on a Reflex save, you get a critical success instead.",
                 "Will": "When you roll a success on a Will save, you get a critical success instead."
             },
+            "SaveCriticalFailureToFailure": {
+                "WillFailureHalfDamage": "When you roll a critical failure on a Will save, you get a failure instead. When you fail a Will save against an effect that deals damage, you take half damage."
```

Comment:
Given how the critical success ones are key'd, perhaps the key for this one should be Will?

Reply:
Still need the half damage part in there. Maybe as a sub-key?

Reply:
Failure to half damage is fairly common I think so we can make a key for that then just do reflex, will, etc as subkeys